test/sys/test_epoll.rs

```diff
@@ -1,24 +1,39 @@
+use std::os::unix::io::AsRawFd;
+
+use nix::{Error, Errno};
 use nix::sys::epoll::{EpollCreateFlags, EpollOp, EpollEvent};
 use nix::sys::epoll::{EPOLLIN, EPOLLERR};
 use nix::sys::epoll::{epoll_create1, epoll_ctl};
-use nix::{Error, Errno};
+use nix::sys::signal::SigSet;
+use nix::sys::signalfd::SignalFd;
+use nix::unistd::close;
 
 #[test]
 pub fn test_epoll_errno() {
+    // Create a new signalfd to use for testing. This is better than using stdin or other such fd.
+    let sfd = SignalFd::new(&SigSet::empty()).unwrap();
```

Comment:
Does it matter? Is `signalfd` overkill or would you just rather reduce cross-dependencies between tests? [This](https://stackoverflow.com/questions/8057892/epoll-on-regular-files#8057973) seems to suggest you can use it on regular files, so let me give that a shot.